ignition_patches/bootstrap/03_ironic.json

```diff
@@ -0,0 +1 @@
+[{"op": "add", "path": "/storage/files/-", "value": {"filesystem": "root", "path": "/etc/sysconfig/network-scripts/ifcfg-eth1", "user": {"name": "root"}, "contents": {"source": "data:text/plain;charset=utf-8;base64,REVWSUNFPWV0aDEKT05CT09UPXllcwpUWVBFPUV0aGVybmV0CkJPT1RQUk9UTz1zdGF0aWMKSVBBRERSPTE3Mi4yMi4wLjEKTkVUTUFTSz0yNTUuMjU1LjI1NS4wCg==", "verification": {}}, "mode": 420}}, {"op": "add", "path": "/systemd/units/-", "value": {"contents": "[Unit]\nDescription=Run Ironic baremetal provisioning\nWants=network-online.target\nAfter=network-online.target\nConditionPathExistsGlob=|/home/core/redhat-coreos-maipo-*.qcow2\n\n[Service]\nEnvironmentFile=/etc/ironicservice\nExecStartPre=-/usr/bin/podman rm ironicservice\nExecStart=/usr/bin/podman run --name ironic --net host --privileged -v /home/core/dnsmasq.conf:/etcd/dnsmasq.conf -v /home/core/${RHCOS_IMAGE_FILENAME_OPENSTACK}:/var/www/html/images/${RHCOS_IMAGE_FILENAME_OPENSTACK} ${IRONIC_IMAGE}\nExecReload=-/usr/bin/podman stop ironicservice\nExecReload=-/usr/bin/podman rm ironicservice\nRestart=always\nRestartSec=30\n", "enabled": false, "name": "ironic.service"}}]
```

Comment:
nitpick, can be fixed later: pull image in PreStart so that `06_deploy_masters` would wait for this service to become active